docs/specifications/cva6_requirement_specification.adoc

```diff
@@ -0,0 +1,391 @@
+Unless required by applicable law or agreed to in writing, any work distributed under the License is distributed on an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+== Introduction
+
+CVA6 is a RISC-V compatible application processor core that can be configured as a 32- or 64-bit core (RV32 or RV64). It includes L1 caches, optional MMU, optional PMP and optional FPU.
+
+It is an industrial evolution of ARIANE created by ETH Zürich and the University of Bologna. It is written in SystemVerilog and maintained by the OpenHW Group.
+
+This specification is organized as requirements that apply to the “Scope of the IP”.
+
+The requirement list is to be approved by the OpenHW Group Technical Work Group (TWG), as well as its change requests.
+
+The specification will be complemented by a user’s guide.
+
+Revision 0 refers to the product of the first CVA6 project led at OpenHW Group. It is a placeholder in case of future evolutions after project freeze (PF gate).
+
+A list of abbreviations is available at the end of this document.
+
+== Scope
+
+=== Scope of the IP
+
+The *scope of the IP* is the subsystem that is specified below and that will undergo verification with a 100% coverage goal. In the verification plans, the scope of the IP can be broken down in several DUT (design under test).
+
+The scope of the IP is the *CVA6 hardware* supporting all the features used in products based on CVA6.
+
+CVA6 exists in two main configurations: *CV64A6* and *CV32A6*. A requirement referring to CVA6 applies to both configurations.
+
+image::images/cva6_scope.png[]
+
+As displayed in the picture above, the IP comprises:
+
+* The CVA6 core;
+* L1 write-through cache;
+* Optional FPU;
+* Optional MMU;
+* Optional PMP;
+* CSR;
+* Performance counters;
+* Interface with the P-Mesh coherence system of OpenPiton.
+
+These are not part of the IP (several solutions can be used):
+
+* CLINT or PLIC Interrupt modules;
+* Debug module (such as DTM);
+* Support of L1 write-back cache (this might come later as an update).
+
+In addition to these main configurations, several fine grain parameters are available.
+
+Unless otherwise stated, an optional feature is controlled by a SystemVerilog parameter. If not selected, the optional feature will not be present in the netlist after synthesis.
+
+The reader’s attention is drawn to the difference between an optional feature (“...shall support as an option...”) and a desired goal (“...should support...”, “....should reduce latency...”).
+
+These are not in the scope of this specification:
+
+* SW layers, such as compiler and OSes (that can however be part of the OpenHW Group CVA6 project);
```

Comment:
It isn't clear to me what the SW projects are part of this project, which is already of very large scope. Wouldn't it be easier to manage well defined project(s) for the compilers and linux/OS support?

Reply:
This is a requirement specification for the core HW base (the object to verify) and it does not touch the project workplan. For more information about SW tasks, refer to the PA gate workplan: https://github.com/openhwgroup/core-v-docs/blob/master/program/Project-Descriptions-and-Plans/CVA6/CVA6-plan-approved-2022.md

Reply:
ok, I was referring to "compiler and OSes (that can however be part of the OpenHW Group CVA6 project)." But you are correct, we can deal with that in the PA workplan.